### ISSUE TITLE
Gmail and contribution box not appearing 

### ISSUE THREAD

**Issue by rohan472000** [rohan472000/rohan472000#11]
@platvin , after merging recent contribution in jokes.py, I tried to include your previous line 

`contents = [line for line in contents if "![Funny Meme]" not in line and line != "\n"]`

But after that gmail and contribution box is not appearing.

Help me on this.

**Comment by platvin** [comment]
Sure, let me take a look.

**Comment by rohan472000** [comment]
@SilentJMA please look into this issue.

**Comment by SilentJMA** [comment]
`contents = [line for line in contents if "![Funny Meme]" not in line and line != "\n"]`
This code uses the in operator to check if the string "![Funny Meme]" is present in each line.
It also checks if the line is not equal to "\n" (an empty line). This condition ensures that empty lines are removed as well.
It uses a combination of not in and != to filter out lines containing "![Funny Meme]" and empty lines.

but for: `contents = [line for line in contents if not re.match(r"!\[Funny Meme\]\(.*\)", line)]`

This code uses the `re.match()` function from the `re` module to match each line against a regular expression pattern.
The regular expression pattern "!\[Funny Meme\]\(.*\)" is used to match lines containing the Markdown image link format for "Funny Meme."
Lines that match the pattern (i.e., contain the image link) are filtered out because of the not operator.
In summary, the primary difference is in the approach to identifying and filtering lines. The first snippet uses a simple substring check with in, while the second snippet uses regular expressions (re.match()) to match a specific pattern. 

I will get back to you and check what happens, my branch is [86 commits behind]

**Comment by rohan472000** [comment]
Sure @SilentJMA

**Comment by SilentJMA** [comment]
you were just missing an empty line between the badge, check it now

**Comment by platput** [comment]
We need at least one new line right before the gmail markdown, the same goes for github banner markdowns. In the current code all the empty lines are being removed I believe and that causes this issue. I had done a fix for this in the last PR I created, may be that was removed in the latest merge. Will check it soon.

**Comment by rohan472000** [comment]
@SilentJMA , not resolved by your PR.

**Comment by SilentJMA** [comment]
sorry my mistake check this one, i deleted the wrong one @rohan472000

**Comment by rohan472000** [comment]
Nope ...still it is not getting resolved.

**Comment by SilentJMA** [comment]
in my fork everything is working fine, please @rohan472000 add empty line after line 5,6 and 12

**Comment by platput** [comment]
After editing the readme file, when you run the script, I believe those new lines are getting removed.

**Comment by rohan472000** [comment]
I removed it..it worked but after running workflow it became same.

**Comment by SilentJMA** [comment]
i changed the joke.py so it can change only the first line containing the joke and leave everything else in the readme file.
check it in the pull request and please add empty line after line 5,6 and 12.
@rohan472000

**Comment by rohan472000** [comment]
Thanks @SilentJMA  it's working now but I will close this issue after sometime in case if it fails I need to discuss here.

Appreciating @platput  and @platvin for looking into it.
